### PR TITLE
Adds Rio Grande do Sul data sources

### PR DESCRIPTION
**Pull request by jeffersoncardoso**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,11 @@ To the extent possible under law, all contributors have waived all copyright and
 
 - [Dados abertos do Rio Grande do Sul](http://dados.rs.gov.br) &mdash; Dados de orgãos púplicos estaduais.
 - [Dados Prefeitura de Porto Alegre](http://datapoa.com.br) &mdash; Dados de orgãos e entidades de administração pública municipal.
+- [Dados abertos TCE-RS](http://dados.tce.rs.gov.br/) &mdash; Dados abertos do tribunal de contas do estado.
```
Comment (juliohm) [inline README.md]: Please remove the trailing `/` in the URL

File: README.md [hunk 2]
```diff
@@ -131,6 +131,11 @@ To the extent possible under law, all contributors have waived all copyright and
...
+- [VisualizaDEE](http://visualiza.dee.planejamento.rs.gov.br/) &mdash; Aplicativos de visualização de dados do Departamento de Economia e Estatística do RS.
```
Comment (juliohm) [inline README.md]: Can you confirm that this website provides open data besides the visualizations?

File: README.md [hunk 3]
```diff
@@ -131,6 +131,11 @@ To the extent possible under law, all contributors have waived all copyright and
...
+- [Receita Dados](http://receitadados.fazenda.rs.gov.br/) &mdash; Dados produzidos pela Receita Estadual do RS
```
Comment (juliohm) [inline README.md]: Trailing '/'

**Review by juliohm — CHANGES_REQUESTED** [review]
Thank you for the additions! :100:

File: README.md (hunk 2 above)
Comment (jeffersoncardoso) [reply README.md]: Yes. The visualizations use Shiny (https://shiny.rstudio.com/) and inside each one there is a link "Download dos Dados" with open data download.

File: README.md (hunk 1 above)
Comment (jeffersoncardoso) [reply README.md]: Solved

File: README.md (hunk 3 above)
Comment (jeffersoncardoso) [reply README.md]: Solved

**Pull request merged**